### PR TITLE
Backend change routing

### PR DESCRIPTION
**Pull request by melyxlin**

### CODE REVIEW

File: web/api/backend.go [hunk 1]
```diff
@@ -111,15 +114,22 @@ func OkHandler(w http.ResponseWriter, r *http.Request) {
...
 
-	http.HandleFunc("/notoken", NoTokenHandler)
+	router.HandleFunc("/notoken", NoTokenHandler)
 
-	if err := http.ListenAndServe(":8081", nil); err != nil {
-		log.Fatal(err)
+	srv := &http.Server{
+		Handler: router,
+		Addr:    "127.0.0.1:8081",
+		// Good practice: enforce timeouts for servers you create!
```
Comment (yousefa00) [inline web/api/backend.go]: Don't forget to remove this comment

**Review by yousefa00 — APPROVED** [review]
Approved, just one leftover comment 🚀

File: web/api/backend.go (hunk 1 above)
Comment (melyxlin) [reply web/api/backend.go]: Done

**Review by leorcodes — APPROVED** [review]
👍

**Pull request merged**